.vib/zipkin/goss/goss.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+http:
+  http://127.0.0.1:{{ .Vars.containerPorts.http }}/api/v2/services:
+    status: 200
+    body:
+      - /yelp-main/
+  http://zipkin:{{ .Vars.service.ports.http }}/api/v2/services:
+    status: 200
+    body:
+      - /yelp-main/
+  http://127.0.0.1:{{ .Vars.containerPorts.http }}/prometheus:
+    status: 200
+    body:
+      - /armeria_server/
+  http://zipkin:{{ .Vars.service.ports.http }}/prometheus:
+    status: 200
+    body:
+      - /armeria_server/
+
+command:
+  {{- $uid := .Vars.containerSecurityContext.runAsUser }}
+  {{- $gid := .Vars.podSecurityContext.fsGroup }}
+  check-user-info:
+    # The UID and GID should always be either the one specified as vars (always a bigger number that the default)
+    # or the one randomly defined by openshift (larger values). Otherwise, the chart is still using the default value.
+    exec: if [ $(id -u) -lt {{ $uid }} ] || [ $(id -G | awk '{print $2}') -lt {{ $gid }} ]; then exit 1; fi
+    exit-status: 0
+  check-configured-env-vars:
+    exec: env
+    exit-status: 0
+    stdout:
+      - /AUTOCOMPLETE_TTL.*{{ .Vars.configOverrides.AUTOCOMPLETE_TTL }}/
+      {{- if not .Vars.usePasswordFile }}
+      # The secret configuration should be set via env
+      - /QUERY_NAMES_MAX_AGE.*{{ .Vars.secretConfigOverrides.QUERY_NAMES_MAX_AGE }}/
+      {{- end }}
```

Comment:
We're hardcoding `usePasswordFile` to `true` in `runtime-parameters.yaml`? Idk if it's worth to support both styles.